package.json

```diff
@@ -6,7 +6,11 @@
     "devDependencies": {
         "@types/jest": "^27.0.1",
         "@types/lodash": "^4.14.172",
+        "@typescript-eslint/eslint-plugin": "^4.31.1",
```

Comment:
Do you know why your editor formats `package.json` for indentation of 4 spaces? My editor formats as 2 spaces, which seems to be the default indentation across the repo.

Reply:
Ahh I had this [extension](https://marketplace.visualstudio.com/items?itemName=lonefy.vscode-JS-CSS-HTML-formatter) installed for some reason. It seemed to ignore my formatter selection and always format on save?

Reply:
Fixed in eb1fe26